README.md

```diff
@@ -1,6 +1,6 @@
-# Stan.js - STAN Server Node.js Client
+# Stan.js - NATS Streaming Server Node.js Client
```

Comment:
I would remove `Server` from this line. The meaning should be that this is a Node.js client for NATS Streaming